### PR TITLE
Draft: Check feasibility for indicator constraints and round properly

### PR DESCRIPTION
**Pull request by dhendryc**
In `is_integer_feasible`, add a check whether indicator constraints are also met (if they are present).

If indicators are present, the rounding process has to be adjusted.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -9,13 +9,13 @@ DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 FrankWolfe = "f55ce6ea-fdc5-4628-88c5-0087fe54bd30"
+HiGHS = "87dc4568-4c63-4d18-b0c0-bb2238e4078b"
```
Comment (matbesancon) [inline Project.toml:12]: do we need highs as main dependency? @dhendryc

File: test/indicator_test.jl [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    end
+
+    x = [0.5, 1.0, 0.75, 0.0, 0.9, 1.0, 1.0, 1.0, 0.0, 0.0]
+    y = [0.0, 0.0, 0.5, 1.0, 0.0, 1.0, 1.0, 0.0, 0.0, 0.0]
+
+    @test Boscia.is_indicator_feasible(o, x) == false
+    @test Boscia.is_indicator_feasible(o, y) == true
+    ind_rounding(x)
+    @test Boscia.is_indicator_feasible(o, x) == true 
+end
```
Comment (matbesancon) [inline test/indicator_test.jl:63]:
```suggestion
end

```

File: Project.toml (hunk 1 above)
Comment (dhendryc) [reply Project.toml:12]: I think it is used the strong branching?

Comment (matbesancon) [reply Project.toml:12]: ah yes indeed. It should be in the project toml already, strange

**Pull request merged**